packages/auto-label/src/auto-label.ts

```diff
@@ -211,22 +236,128 @@ handler.autoDetectLabel = (
   )?.github_label;
 };
 
-const BACKFILL_LABEL = 'auto-label:backfill';
+handler.mainLogic = async function mainLogic(
```

Comment:
Is there a more descriptive name for this function? `handler` also has a decent amount of logic, so it's not clear from the names.

Reply:
Good point! Essentially, the handler function deals with the different logic that can trigger the main function, which is eventually adding a label to the repo and/or issue. I added a different title, hopefully it's a little better :)